django/contrib/admin/static/admin/js/nav_sidebar.js

```diff
@@ -2,18 +2,37 @@
 {
     const toggleNavSidebar = document.getElementById('toggle-nav-sidebar');
     if (toggleNavSidebar !== null) {
+        const navLinks = document.querySelectorAll('#nav-sidebar a');
+
+        function disableNavLinkTabbing() {
+            for (const navLink of navLinks) {
+                navLink.setAttribute("tabindex", "-1");
```

Comment:
`tabindex` is an element property

```suggestion
                navLink.tabIndex = -1;
```

https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/tabIndex

Reply:
Thanks, got those.